src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  background-color: $white;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $grey;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin: 40px 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $black;
+  }
+
+  &__code {
+    font-family: $roboto;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $dark-grey;
+  }
+
+  &__reviews {
+    font-family: $roboto;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $black;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+    align-items: end;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+    margin-bottom: 16px;
+
+    &-title {
+      font-family: $roboto;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $dark-grey;
+    }
+
+    &-value {
+      font-family: $roboto;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $black;
+    }
+  }
+
+  &__button {
+    display: block;
+    margin: 0 auto;
+    font-family: $roboto;
+    text-transform: uppercase;
+    font-size: 14px;
+    color: $white;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+
+    border: none;
+    outline: none;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+
+    background: $blue;
+
+    &:hover {
+      background-color: $white;
+      color: $blue;
+      border: 1px solid $blue;
```

Comment:
It is a good practice to add a transparent border for the default state and just change its color on hover